ublox/src/ubx_packets/types.rs

```diff
@@ -89,3 +89,27 @@ impl<'a> TryFrom<&NavPosVelTimeRef<'a>> for DateTime<Utc> {
         Ok(DateTime::from_utc(dt, Utc))
     }
 }
+
+pub(crate) struct FieldIter<I>(pub(crate) I);
```

Comment:
Was able to get rid of ubx_iter, seems a little simpler to avoid more code gen

Reply:
nooooo my beautiful macro